### PR TITLE
feat: Add docs for live sessions

### PR DESCRIPTION
**Pull request by filipelima18**
Preview [here](https://2t92qr-3000.csb.app/learn/devboxes/live-sessions).

Closes PC-1689

### CODE REVIEW

File: packages/projects-docs/pages/learn/devboxes/live-sessions.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import { Callout } from 'nextra-theme-docs'
+
+# Live Sessions
+
+## What are live sessions?
+
+Live sessions enable the creation of a temporary open session in a Devbox, which allows you to invite any user with a CodeSandbox account to join the Devbox and collaborate live. A live session can include unlimited users.
+
+The key difference between live sessions and the default live collaboration of Devboxes is that live sessions don't require invited users to be part of the workspace to which the Devbox belongs.
```
Comment (danilowoz) [inline packages/projects-docs/pages/learn/devboxes/live-sessions.mdx:16]: Good one!

**Review by danilowoz — APPROVED** [review]
Thank you!!!

**Review by danilowoz — APPROVED** [review]
Thank you!!!

**Review by necoline — APPROVED** [review]
Nice work!

File: packages/projects-docs/styles.css [hunk 2]
```diff
@@ -101,17 +101,6 @@ aside {
 /* General colors
   ------------------------ */
 
-html[class~="light"] body,
```
[the remark this answers is not in the document]
Comment (filipelima18) [reply packages/projects-docs/styles.css:104]: Thanks!

**Pull request merged**